index.html

```diff
@@ -1121,14 +1183,21 @@ <h1 style="font-size: 48px;">My headline</h1>
 
 						---
 
-						### Exercise
+						### Borders
 
-						- Try adding the following properties to your project divs
-							- Padding
-							- Margin
-							- Border
+						- You can set the border to have various styles
```

Comment:
This is small--and not vital, so feel free to ignore :) --but for slides with only a single bullet, we can probably remove the list style entirely to simplify the content on the page! I only noticed on these several slides.